cpp/plugins/cucim.kit.cuslide/src/cuslide/cuslide.cpp

```diff
@@ -100,42 +100,32 @@ static bool CUCIM_ABI parser_parse(CuCIMFileHandle* handle, cucim::io::format::I
 
     auto tif = static_cast<cuslide::tiff::TIFF*>(handle->client_data);
 
-    std::vector<size_t> main_ifd_list;
-
     size_t ifd_count = tif->ifd_count();
     size_t level_count = tif->level_count();
-    for (size_t i = 0; i < ifd_count; i++)
-    {
-        const std::shared_ptr<cuslide::tiff::IFD>& ifd = tif->ifd(i);
-
-        //        const char* char_ptr = ifd->model().c_str();
-        //        uint32_t width = ifd->width();
-        //        uint32_t height = ifd->height();
-        //        uint32_t bits_per_sample = ifd->bits_per_sample();
-        //        uint32_t samples_per_pixel = ifd->samples_per_pixel();
-        uint64_t subfile_type = ifd->subfile_type();
-        //        printf("image_description:\n%s\n", ifd->image_description().c_str());
-        //        printf("model=%s, width=%u, height=%u, model=%p bits_per_sample:%u, samples_per_pixel=%u, %lu \n",
-        //        char_ptr,
-        //               width, height, char_ptr, bits_per_sample, samples_per_pixel, subfile_type);
-        if (subfile_type == 0)
-        {
-            main_ifd_list.push_back(i);
-        }
-    }
 
-    // Assume that the image has only one main (high resolution) image.
-    if (main_ifd_list.size() != 1)
+    if (tif->ifd(0)->image_description().rfind("Aperio", 0) == 0)
     {
-        throw std::runtime_error(
-            fmt::format("This format has more than one image with Subfile Type 0 so cannot be loaded!"));
+        // Aperio SVS format
```

Comment:
Is there something else expected in this block? If not, should we consolidate the `if`/`else` here into a single `if`?